angular.json

```diff
@@ -191,4 +191,4 @@
     }
   },
   "defaultProject": "mifosx-web-app"
-}
+}
```

Comment:
@Abhirup-99 There is no difference but still it is showing changes at this line.
How can I fix this?

Reply:
Don't commit this file.

Reply:
resolved👍

Thanks, @Abhirup-99